packages/vue/src/components/atoms/SfBullets/SfBullets.stories.js

```diff
@@ -20,7 +20,12 @@ export default {
         category: "Props",
       },
     },
-    click: { action: "set Value of current", table: { category: "Events" } },
+    // click: { action: "set Value of current", table: { category: "Events" } },
+  },
+  parameters: {
+    actions: {
+      handles: ['click', '.sf-bullet']
```

Comment:
should be there any info about this click in the actions tab? I can't see anything :)

Reply:
I tried to implement it to make them work and show in action tab at the same time, but it doesn't work. I remove it.